smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy

```diff
@@ -120,7 +146,65 @@ use smithy.test#httpResponseTests
             doubleValue: "-Infinity",
             floatValue: "-Infinity"
         }
-    }
+    },
+    {
+        id: "RpcV2CborSupportsIndefiniteLengthStrings",
+        protocol: rpcv2Cbor,
+        documentation: "Supports indefinite length text strings.",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+            "Content-Type": "application/cbor"
+        }
+        method: "POST",
+        bodyMediaType: "application/cbor",
+        uri: "/service/RpcV2Protocol/operation/SimpleScalarProperties",
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=oWtzdHJpbmdWYWx1ZX94HUFuIGV4YW1wbGUgaW5kZWZpbml0ZSBzdHJpbmcscSBjaHVua2VkIG9uIGNvbW1h%2Fw%3D%3D
+        body: "oWtzdHJpbmdWYWx1ZX94HUFuIGV4YW1wbGUgaW5kZWZpbml0ZSBzdHJpbmcscSBjaHVua2VkIG9uIGNvbW1h/w=="
+        params: {
+            stringValue: "An example indefinite string, chunked on comma"
+        }
+    },
+    {
+        id: "RpcV2CborSupportsIndefiniteLengthByteStrings",
```

Comment:
This test doesn't really make sense, how is the implementation supposed to know to encode this way?

Reply:
Also, how is the implementation supposed to know to encode as a byte string? It's modeled as `string`.

Reply:
Good point. I have now added a `blobValue: Blob` field to the structure. The test for indefinite byte strings now utilizes that.